psc-docs/Ctags.hs

```diff
@@ -1,5 +1,6 @@
 module Ctags (dumpCtags) where
 
+import Prelude
```

Comment:
It looks like changing:
```
	modified:   psc-docs/Ctags.hs
	modified:   psc-docs/Etags.hs
	modified:   psc-docs/Main.hs
	modified:   psc-docs/Tags.hs
	modified:   tests/TestPscIde.hs
	modified:   tests/TestPscPublish.hs
```

Makes ghcid work.

I can probably just do that locally. Should I remove the `import Prelude` statements?